coursemapper-kg/recommendation/app/services/course_materials/db/neo4_db.py

```
@@ -3,6 +3,7 @@
 import re
 import logging
 from log import LOG
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)